docs/import/list_ee_assets.js

```diff
@@ -10,31 +10,12 @@ export {getDisasterAssetsFromEe, getStatesAssetsFromEe};
  *     retrieved assets in the form [['WA', {'asset/path': LayerType,...}], ...]
  */
 function getStatesAssetsFromEe(states) {
```

Comment:
As a follow-up, we could fetch all disasters' states' data eagerly, instead of doing it on demand.

Reply:
Added a TODO